app/stories/pages/eligibility.stories.tsx

```diff
@@ -1,14 +1,44 @@
+import { useState } from '@storybook/client-api'
 import { ComponentMeta, ComponentStory } from '@storybook/react'
 
 import EligibilityPage from '@pages/eligibility'
 
+import { getBackRoute, getForwardRoute } from '@utils/routing'
+
+import {
+  getEmptyMockSession,
+  getMockSessionData,
+} from '../../tests/helpers/mockData'
+
 export default {
   title: 'Pages/Eligibility',
   component: EligibilityPage,
+  argTypes: {
+    session: {
+      table: {
+        disable: true,
+      },
+    },
+  },
 } as ComponentMeta<typeof EligibilityPage>
 
-const Template: ComponentStory<typeof EligibilityPage> = () => (
-  <EligibilityPage previousRoute="review" />
-)
+const Template: ComponentStory<typeof EligibilityPage> = (args) => {
+  const [session, setSession] = useState(args.session)
+  args.session = session
+  args.setSession = setSession
+  args.backRoute = getBackRoute('/eligibility', session)
+  const reviewMode = args.reviewMode === undefined ? false : args.reviewMode
+  args.forwardRoute = getForwardRoute('/eligibility', reviewMode, session)
+  return <EligibilityPage {...args} />
+}
+
+export const Default = Template.bind({})
+Default.args = {
+  session: getEmptyMockSession(),
+}
 
-export const Eligibility = Template.bind({})
+export const Update = Template.bind({})
+Update.args = {
+  session: getMockSessionData(),
+  reviewMode: true,
+}
```

Comment:
Notes for reviewers: For editable form pages, we show 2 stories in storybook:

1. The blank form you would see when you first navigate to that page
2. The filled out form you would see when you have gone all the way through the form wizard and want to go back and edit this particular page